data/catalogs/test_data_catalog.py

```diff
@@ -0,0 +1,117 @@
+"""A command line script for testing predefined data catalogs.
+
+This script allows for testing separate data catalog items and the complete data catalog.
+When testing the data catalog all the paths of the data catalog sources are
+checked if they exist.
+Call for testing the data catalog:
+    'python test_data_catalog.py path/to/datacatalog.yml'
+
+Separate data catalog sources can be tested by giving the dataset source name as
+an extra argument. The argument should be preceded by -ds. It is also possible to specify
+the version of the dataset source by supplying the call with a -dsv flag and version
+number. The dataset is opened and if the dataset contains geo data the dataset is clipped
+to a small bounding box surrounding Deltares Delft office.
+Example dataset source test call:
+    'python test_data_catalog.py path/to/datacatalog.yml -ds chelsa -dsv 1.2
+    -r "{'bbox':[4.333409,51.962159,4.42835,52.006873]}"'
```

Comment:
if you're using multiple characters it's customary to use `--` for your flags and only `-` if you use one character. (e.g. `-d` but `--dsv`)